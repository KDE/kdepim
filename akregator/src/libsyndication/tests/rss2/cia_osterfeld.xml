<?xml version="1.0"?>
<!--
    Real-world use of <cloud>
   -->
<rss version="2.0"><channel><title>osterfeld</title><link>http://cia.navi.cx/stats/author/osterfeld</link><description>CIA Stats</description><cloud path="/RPC2" domain="cia.navi.cx" protocol="xml-rpc" port="80" registerProcedure="stats.subscribe.rss2" /><item><pubDate>Fri, 17 Mar 2006 13:53:27 +0000</pubDate><guid>http://cia.navi.cx/stats/author/osterfeld/.message/65e7a</guid><link>http://cia.navi.cx/stats/author/osterfeld/.message/65e7a</link><description>         &lt;div style="border: 1px solid #888; background-color: #DDD; padding: 0.25em 0.5em; margin: 0em;"&gt;              Commit by &lt;strong&gt;osterfeld&lt;/strong&gt;                          &lt;span style="color: #888;"&gt; :: &lt;/span&gt;                          r&lt;b&gt;519554&lt;/b&gt;             &lt;b&gt;yakuake&lt;/b&gt;/trunk/extragear/utils/yakuake/src/main_window.cpp:                      &lt;/div&gt;         &lt;div style="padding: 0em; margin: 0.5em 0em;"&gt;             fix case in menu items and action names         &lt;/div&gt;     </description><title>fix case in menu items and action names</title></item>
<channel>
</rss>