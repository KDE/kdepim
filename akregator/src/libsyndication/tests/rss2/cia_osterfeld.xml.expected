# Feed begin ######################
title: #osterfeld#
link: #http://cia.navi.cx/stats/author/osterfeld#
description: #CIA Stats#
# Item begin ######################
id: #http://cia.navi.cx/stats/author/osterfeld/.message/65e7a#
title: #fix case in menu items and action names#
link: #http://cia.navi.cx/stats/author/osterfeld/.message/65e7a#
description: #<div style="border: 1px solid #888; background-color: #DDD; padding: 0.25em 0.5em; margin: 0em;"> Commit by <strong>osterfeld</strong> <span style="color: #888;"> :: </span> r<b>519554</b> <b>yakuake</b>/trunk/extragear/utils/yakuake/src/main_window.cpp: </div> <div style="padding: 0em; margin: 0.5em 0em;"> fix case in menu items and action names </div>#
datePublished: #Fri Mar 17 14:53:27 2006#
dateUpdated: #Fri Mar 17 14:53:27 2006#
# Item end ########################
# Feed end ########################

